plugins/indexing/plugin.go

```diff
@@ -93,14 +102,26 @@ func main() {
 		},
 	})
 	if err != nil {
-		panic(err)
+		abort(err)
 	}
 	std.RegisterInterfaces(interfaceRegistry)
 	app.ModuleBasics.RegisterInterfaces(interfaceRegistry)
 
+	logFile, err := log.GetLogFile()
+	if err != nil {
+		abort(err)
+	}
+	defer logFile.Close()
```

Comment:
Perhaps this can be moved up so that we can use the logger to write errors from codec registration.